exo-go/src/aws_helpers/secrets_helpers.go

```diff
@@ -0,0 +1,165 @@
+package awsHelper
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"io"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/s3"
+)
+
+const secretsFile string = "secrets.tfvars"
+
+// CreateSecretsStore creates an S3 bucket  and file object used for secrets management
+func CreateSecretsStore(secretsBucket, region string) error {
+	s3client := createS3client(region)
+
+	hasBucket, err := hasBucket(s3client, secretsBucket)
+	if err != nil {
+		return err
+	}
+
+	// create bucket if it doesn't exist
+	if !hasBucket {
+		_, err = s3client.CreateBucket(&s3.CreateBucketInput{Bucket: &secretsBucket})
+		if err != nil {
+			return err
+		}
+	}
```

Comment:
Can the logic for creating a bucket if it doesn't exist be extracted into its own function?